### PR TITLE
Adding API for company_model

### PR DESCRIPTION
**Pull request by Anntroy**

### CODE REVIEW

File: data_ocean/serializers/company_serializers.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+from data_ocean.models.common_models import Authority, Status, TaxpayerType
+from data_ocean.models.company_models import Bylaw, CompanyType, Company, Assignee, BancruptcyReadjustment, CompanyDetail, CompanyToKved, ExchangeDataCompany, FounderFull, Predecessor, CompanyToPredecessor, Signer, TerminationStarted
+from data_ocean.models.kved_models import Kved
+from data_ocean.models.main import DataOceanModel
+from data_ocean.models.ruo_models import State
+
+from rest_framework import serializers
+from simple_history.models import HistoricalRecords
```
Comment (criticus) [inline data_ocean/serializers/company_serializers.py:8]: is this needed?

File: data_ocean/serializers/company_serializers.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+from data_ocean.models.common_models import Authority, Status, TaxpayerType
+from data_ocean.models.company_models import Bylaw, CompanyType, Company, Assignee, BancruptcyReadjustment, CompanyDetail, CompanyToKved, ExchangeDataCompany, FounderFull, Predecessor, CompanyToPredecessor, Signer, TerminationStarted
+from data_ocean.models.kved_models import Kved
+from data_ocean.models.main import DataOceanModel
```
Comment (criticus) [inline data_ocean/serializers/company_serializers.py:4]: it's unused

File: data_ocean/serializers/company_serializers.py (hunk 1 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: I've cleaned up all, that I don't need.

File: data_ocean/serializers/company_serializers.py (hunk 2 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: I leave the few I really need

File: data_ocean/serializers/company_serializers.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class CompanySerializer(serializers.ModelSerializer):
+    company_type = serializers.CharField(max_length=100)
+    bylaw = serializers.CharField(max_length=100)
+    state = serializers.CharField(max_length=100)
+    authority = serializers.CharField(max_length=500)
+    parent = serializers.CharField(max_length=500)
+    class Meta:
+        model = Company
+        fields = ('id', 'name', 'short_name', 'company_type', 'edrpou', 'address', 'state', 'bylaw', 'registration_date', 'registration_info', 'contact_info', 'authority', 'parent', 'history')
```
Comment (criticus) [inline data_ocean/serializers/company_serializers.py:25]: do we need history field?

Comment (criticus) [reply data_ocean/serializers/company_serializers.py:25]: did you remove exchange_data, predecessors and others completely? I meant to move them inside company

File: data_ocean/serializers/company_serializers.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        fields = ('id', 'name')
+
+class CompanyTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CompanyType
+        fields = ('id', 'name')
+
+class CompanySerializer(serializers.ModelSerializer):
+    company_type = serializers.CharField(max_length=100)
+    bylaw = serializers.CharField(max_length=100)
```
Comment (criticus) [inline data_ocean/serializers/company_serializers.py:19]: does 100 enough for bylaw?

Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: Maybe not, it's the same as in company_model.

File: data_ocean/serializers/company_serializers.py (hunk 1 above)
Comment (criticus) [reply data_ocean/serializers/company_serializers.py:3]: where do you use HistoricalRecords?

File: data_ocean/serializers/company_serializers.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        model = CompanyType
+        fields = ('id', 'name')
+
+class CompanySerializer(serializers.ModelSerializer):
+    company_type = serializers.CharField(max_length=100)
+    bylaw = serializers.CharField(max_length=300)
+    state = serializers.CharField(max_length=100)
+    authority = serializers.CharField(max_length=500)
+    parent = serializers.CharField(max_length=500)
+    assignee = serializers.CharField(max_length=100)
```
Comment (criticus) [inline data_ocean/serializers/company_serializers.py:21]: why don't you use `AssigneeSerializer` as it's defined below?

File: data_ocean/serializers/company_serializers.py (hunk 1 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: I've fix it

File: data_ocean/serializers/company_serializers.py (hunk 3 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: done

File: data_ocean/serializers/company_serializers.py (hunk 4 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: I've changed it to 300

File: data_ocean/serializers/company_serializers.py (hunk 5 above)
Comment (Anntroy) [reply data_ocean/serializers/company_serializers.py]: done

**Pull request merged**